cfgov/jinja2/v1/_includes/atoms/select.html

```diff
@@ -39,7 +39,7 @@
 {% endif %}
 {% from 'macros/util/format/url.html' import slugify as slugify %}
 {% set id_slug = slugify( value.label ) %}
-<div class="form-l_col form-l_col-{{ 'form-l_col-' ~ value.size if value.size else 'form-l_col-1'}}">
+<div class="form-l_col {{ 'form-l_col-' ~ value.size if value.size else 'form-l_col-1'}}">
```

Comment:
I changed it this way to make it consistent with what @anselmbradford suggested. I like your suggestion of doing `form-l-col-{{value.size ....}}` but can we come to a consensus on what we're changing this pattern to, since it's really common.